app/src/main/java/com/example/calculadoramvp/activity/MainActivity.java

```diff
@@ -4,16 +4,18 @@
 import android.os.Bundle;
 import butterknife.ButterKnife;
 import butterknife.OnClick;
+import butterknife.OnFocusChange;
```

Comment:
remove the **OnFocusChange** you don't use it now

Reply:
done